selenium_tests/util.py

```diff
@@ -0,0 +1,158 @@
+import os
+from base64 import b64encode
+import socket
+from urllib.parse import (
+    ParseResult,
+    urlparse,
+)
+from subprocess import check_call, check_output, DEVNULL
+from django.conf import settings
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+from micromasters.utils import now_in_utc
+
+
+DEFAULT_PASSWORD = 'pass'
+
+
+class Browser:
+    ignored_log_messages = {
+        # TODO: remove
+        'Failed to load resource:',
+        'chrome-extension',
+        'This page includes a password or credit card input in a non-secure context',
+        'favicon.ico',
+        "'webkitURL' is deprecated. Please use 'URL' instead",
+        "zendesk",
+        "__webpack_hmr",
+        "Warning: Accessing PropTypes via the main React package is deprecated.",
+        "Warning: ReactTelephoneInput: React.createClass is deprecated"
+    }
+
+    def __init__(self, driver, live_server):
+        self.driver = driver
+        self.live_server = live_server
+
+    def wait(self, timeout=5):
+        return WebDriverWait(driver=self.driver, timeout=timeout)
+
+    def wait_until_loaded(self, by, value):
+        return self.wait().until(
+            lambda driver: driver.find_element(by, value)
+        )
+
+    def wait_until_element_count(self, by, value, expected_count):
+        return self.wait().until(
+            lambda driver: len(driver.find_elements(by, value)) == expected_count
+        )
+
+    def get_console_errors(self):
+        # TODO: to lowercase for zendesk?
+        return [
+            entry['message'] for entry in self.driver.get_log("browser")
```

Comment:
This will clear the log messages after retrieving them, I'm not sure it should be a getter